tests/drop_packets/drop_packets.py

```diff
@@ -258,6 +261,107 @@ def ports_info(ptfadapter, duthosts, rand_one_dut_hostname, setup, tx_dut_ports)
     return data
 
 
+def acl_setup(duthosts, loganalyzer, template_dir, acl_rules_template, del_acl_rules_template, dut_tmp_dir,
+              dut_clear_conf_file_path):
+    for duthost in duthosts:
+        acl_facts = duthost.acl_facts()["ansible_facts"]["ansible_acl_facts"]
+        if 'DATAACL' not in acl_facts.keys():
+            pytest.skip("Skipping test since DATAACL table is not supported on this platform")
```

Comment:
The skip message is a little confusing. The absence of ```DATAACL``` table dosen't means ```not supported```.  
```suggestion
            pytest.skip("Skipping test since DATAACL table is not present on DUT")
```

Reply:
Fixed